spec/models/game_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'rails_helper'
+
+RSpec.describe Game, type: :model do
+  
+  describe "#populate_board!" do
+    it 'places the correct number of pawns on the board' do
+      
+      white_user = FactoryGirl.create(:user)
+      black_user = FactoryGirl.create(:user)
+
+      # user = User.create(game_id: 1, email: 'dummyemail@123.com', password: 'SecretPassword')
+      game = FactoryGirl.create(:game)
```

Comment:
Can do this in 1 line.
`game = FactoryGirl.create(:game, white_player: white_user, black_player: black_user)`